### PR TITLE
Create security section for new content

### PR DESCRIPTION
**Pull request by rosieyohannan**
@atodd-circleci just tagging you so you can add more content off this branch :-)

### CODE REVIEW

File: jekyll/_cci2/security-recommendations.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+* TOC
+{:toc}
+
+## Checklist for using CircleCI securely as a customer
+{: #checklist-for-using-circleci-securely-as-a-customer }
+
+If you are getting started with CircleCI, there are some security best practices you can ask your team to consider as _users_ of CircleCI:
+
+- Minimise the number of secrets (private keys / environment variables) your
```
Comment (thomasr8) [inline jekyll/_cci2/security-recommendations.md:19]:
```suggestion
- Minimize the number of secrets (private keys / environment variables) your
```

File: jekyll/_cci2/security-recommendations.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  build needs and rotate secrets regularly.
+  - It is important to rotate secrets regularly in your organization, especially as team members come and go.
+  - Rotating secrets regularly means your secrets are only active for a certain amount of time, helping to reduce possible risks if keys are compromised.
+  - Ensure the secrets you _do_ use are of limited scope - with only enough permissions for the purposes of your build. Understand the role and permission systems of other platforms you use outside of CircleCI; for example, IAM permissions on AWS, or GitHub's [Machine User](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users) feature.
+- Sometimes user misuse of certain tools might accidentally print secrets to stdout which will land in your logs. Please be aware of:
```
Comment (thomasr8) [inline jekyll/_cci2/security-recommendations.md:24]:
```suggestion
- Sometimes user misuse of certain tools might accidentally print secrets to `stdout` which will land in your logs. Please be aware of:
```

File: jekyll/_cci2/security-recommendations.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  - running `env` or `printenv` which will print all your environment variables to stdout.
```
Comment (thomasr8) [inline jekyll/_cci2/security-recommendations.md:25]:
```suggestion
  - running `env` or `printenv` which will print all your environment variables to `stdout`.
```

File: jekyll/_cci2/security-recommendations.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  - literally printing secrets in your codebase or in your shell with `echo`.
+  - programs or debugging tools that print secrets on error.
+- Consult your VCS provider's permissions for your organization (if you are in an organizations) and try to follow the [Principle of Least Privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).
```
Comment (thomasr8) [inline jekyll/_cci2/security-recommendations.md:28]:
```suggestion
- Consult your VCS provider's permissions for your organization (if you are in an organization) and try to follow the [Principle of Least Privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).
```

File: jekyll/_cci2/security-recommendations.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- Use Restricted Contexts with teams to share environment variables with a select security group. Read through the [contexts]({{ site.baseurl }}/2.0/contexts/#restricting-a-context) document to learn more.
+- Ensure you audit who has access to SSH keys in your organization.
+- Ensure that your team is using Two-Factor Authentication (2FA) with your VCS ([Github 2FA](https://help.github.com/en/articles/securing-your-account-with-two-factor-authentication-2fa), [Bitbucket](https://confluence.atlassian.com/bitbucket/two-step-verification-777023203.html)). If a user's GitHub or Bitbucket account is compromised a nefarious actor could push code or potentially steal secrets.
```
Comment (thomasr8) [inline jekyll/_cci2/security-recommendations.md:31]:
```suggestion
- Ensure that your team is using Two-Factor Authentication (2FA) with your VCS ([Github 2FA](https://help.github.com/en/articles/securing-your-account-with-two-factor-authentication-2fa), [Bitbucket](https://confluence.atlassian.com/bitbucket/two-step-verification-777023203.html)). If a user's GitHub or Bitbucket account is compromised, a nefarious actor could push code or potentially steal secrets.
```

**Review by thomasr8 — APPROVED** [review]
Looks great! Just some minor comments.

**Comment by rosieyohannan** [comment]
Thanks @thomasr8 !

**Pull request merged**